src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSServiceType.cs

```diff
@@ -17,4 +21,16 @@ internal static bool IsSqsService(string service)
 
     internal static bool IsSnsService(string service)
         => SNSService.Equals(service, StringComparison.OrdinalIgnoreCase);
+
+    internal static bool IsBedrockService(string service)
+        => BedrockService.Equals(service, StringComparison.OrdinalIgnoreCase);
+
+    internal static bool IsBedrockAgentService(string service)
+        => BedrockAgentService.Equals(service, StringComparison.OrdinalIgnoreCase);
```

Comment:
I can't find this method being used?

Reply:
If you plan to split the overall changes into multiple PRs divided by feature, I suggest ensuring each PR accomplishes only the specific tasks intended for it. Please avoid including partial changes meant for future PRs in the current one. :)